tasks/common_packages.yml

```diff
@@ -34,3 +34,11 @@
       - locales
       - nfs-client
       - net-tools
+      - tree
+      - tmux
+      - ncdu
+      - iostat
+      - dialog
+      - nmap
```

Comment:
Why? Do you really think we need this more than one or two times per year? Also why would we need it on every server?